packages/lodestar/src/chain/chain.ts

```diff
@@ -283,8 +283,12 @@ export class BeaconChain implements IBeaconChain {
     if (!fs.existsSync(byDate)) {
       fs.mkdirSync(byDate, {recursive: true});
     }
-    const fileName = `${byDate}/${type}_${suffix}_${Date.now()}.ssz`;
-    fs.writeFileSync(fileName, bytes);
+    const fileName = `${byDate}/${type}_${suffix}.ssz`;
```

Comment:
Do they need to be organized in directories by date? You can use the linux time to see what file is at what date. With this files will still be duplicated 1 per day

Reply:
grouping by date would help investigate issues better imo, we can for sure use linux time to group/count files but it's just more convenient to go to sub-folder and see how many inValidSSZObject files there

yeah there's a chance files are stored end of day `n` and beginning of day `n + 1`, we'll see if it's a lot and cause any issues